inbc-program/inbc/parser/parser.py

```diff
@@ -85,7 +85,7 @@ def parse_source_args(self) -> None:
 
         # Application Remove Command
         app_remove_parser = app_subparsers.add_parser('remove')
-        app_remove_parser.add_argument('--gpgKeyName', '-gkn', required=True,
+        app_remove_parser.add_argument('--gpgKeyName', '-gkn', required=False,
                                        type=lambda x: validate_string_less_than_n_characters(
                                            x, 'str', 50),
                                        help='GPG key name of the source to remove.')
```

Comment:
**Suggestion:** The '--gpgKeyName' argument in the 'remove' command has been changed from required to optional. This could lead to unexpected behavior if the 'application_remove' function expects this argument to always be provided. Consider adding a check to handle the absence of this argument or keep it required if it's essential for the operation. [maintainability]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
        # No change to the code itself, but ensure compatibility with the rest of the codebase.
```

Reply:
already done.